components/prism-bro.js

```diff
@@ -0,0 +1,102 @@
+Prism.languages.bro = {
+    'comment': [
+        {
+	        pattern: /(^|[^\\$])#.*/,
+		            	lookbehind: true
+		            }
+	],
+
+    'string': /(["'])(\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
+
+    'boolean': /\b(T|F)\b/,
+
+    'keyword': [
```

Comment:
You can use a single regex instead of the array here. You should also use a non-capturing group.